### PR TITLE
Add does drugs

### PR DESCRIPTION
**Pull request by awesomeousPrime**
Adding `does_drugs` lab attribute to available data items.

This is going to be used for a test on another project.

### CODE REVIEW

**Comment by healthlabs-bugman** [comment]
Testing file changes were not found.
Paging @fortoulrodolfo to review the front-end work in this pull request!
Paging @Chekote to review the testing work in this pull request!

File: js/findalab.js [hunk 1]
```diff
@@ -1116,6 +1117,7 @@
               .attr('data-country', lab.country)
               .attr('data-fax_number', lab.fax_number)
               .attr('data-network_id', lab.network_id)
+              .attr('data-does_drugs', lab.does_drugs)
```
Comment (Chekote) [inline js/findalab.js]: Please add testing for this new functionality. See `onlyStatesFilter.feature` for an example of how you might do this.

**Comment by awesomeousPrime** [comment]
Closing PR: It looks like I might not need this change after all.

**Comment by awesomeousPrime** [comment]
Just remembered how I used this for testing and is still needed.

**Pull request closed without merging**